janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/rest/RestElasticSearchClient.java

```diff
@@ -133,7 +133,7 @@ public void clusterHealthRequest(String timeout) throws IOException {
     public boolean indexExists(String indexName) throws IOException {
         boolean exists = false;
         try {
-            delegate.performRequest("GET", "/" + indexName);
+            delegate.performRequest("GET", "/" + indexName.toLowerCase());
```

Comment:
It's a store parameter and is used directly without case check in the existing implementation. So I think it's okay not to protect it but if you want to improve what about checking it in the ElasticSearchIndex constructor when it's first read and throw an illegal argument exception if it's not lower case?